### DISCUSSION TITLE
How to remove falsely detected face? (there is no face...)

### DISCUSSION THREAD

**Discussion by bw1faeh0** [immich-app/immich#8541] [Q&A]
Hi,

I have a lot of pictures, where faces were detected, that are no faces. like:
![grafik](https://github.com/immich-app/immich/assets/29235332/b1f2cd29-4319-49c2-9a7c-813d778c0126)

In the details tab I can modify the face, but I haven't found a way to _remove_ a face:
![grafik](https://github.com/immich-app/immich/assets/29235332/bfaa11f2-2f3a-4314-9f05-9bef41057f13)

Any suggestions?

**Comment by bo0tzz** [comment]
This isn't possible at the moment, see #5962.

Have you changed the score/distance thresholds in the ML settings? Detecting faces where there are none should not happen on the default settings.

**Reply by bw1faeh0** [reply]
Yes I lowered the default MIN DETECTION SCORE to 0.4 since immich does not detect a lot of faces. Even in simple portraits the face is not detected.
